src/farm.py

```diff
@@ -33,6 +33,43 @@
 my_mission_record = None
 
 
+class MyAgent:
+    def __init__(self, world_state):
+        self.reward = 0
+        self.captured_sheeps = set()
+        self.world_state = None
+        self.x = 0.5
+        self.z = 0.5
+
+    def isSheepInPen(self, entity):
+        x = entity["x"]
+        z = entity["z"]
+        return x > 1 and x < 5 and z > 1 and z < 5
+
+    def characterMoved(self, entity):
+        x = entity["x"]
+        z = entity["z"]
+        return not (x == self.x and z == self.z)
+
+    def updateReward(self):
+        for entity in self.world_state["entities"]:
```

Comment:
add a quick comment explaining the rewards perhaps (for record keeping when we have more)?  i.e.
```
Rewards:
+100 per captured sheep in pen
-1 for each step agent makes
```